js/playlists.view.js

```diff
@@ -0,0 +1,165 @@
+//
+// Zookeeper Online
+//
+// @author Jim Mason <jmason@ibinx.com>
+// @copyright Copyright (C) 1997-2020 Jim Mason <jmason@ibinx.com>
+// @link https://zookeeper.ibinx.com/
+// @license GPL-3.0
+//
+// This code is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License, version 3,
+// as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License,
+// version 3, along with this program.  If not, see
+// http://www.gnu.org/licenses/
+//
+
+/*! Zookeeper Online (C) 1997-2020 Jim Mason <jmason@ibinx.com> | @source: https://zookeeper.ibinx.com/ | @license: magnet:?xt=urn:btih:1f739d935676111cfff4b4693e3816e664797050&dn=gpl-3.0.txt GPL-v3.0 */
+
+$().ready(function(){
+    // fetch the days with playlists for the selected month and enable the on the calendar.
+    function setAvailableDays(showDate) {
+        let url = '/index.php?action=playlistDaysByDate&viewdate=' + showDate;
+        $.ajax({
+            dataType : 'json',
+            type: 'GET',
+            accept: "application/json; charset=utf-8",
+            url: url,
+        }).done(function (goodDays) {
+            let excludeDays = $('td[data-handler="selectDay"]').filter(function(){
```

Comment:
For days of the month < 10, I am seeing the ajax method return zero-prefixed days (e.g., 03, 07), which are then not found and highlighted in the calendar, as the calendar has 3, 7, etc., for the days (no zero).  Filtering should exclude the leading zero on the returned day, if any, so that it matches the day in the calendar.